hacks/cutNewVersions.sh

```diff
@@ -27,7 +27,7 @@ git commit -m "release: the big commit"
 
 # now start committing the small stuff
 # if there are uncommitted changes to this file, commit it:
-fi [ -n "$(git status --porcelain ./hacks/cutNewVersions.sh)" ]; then
+if [ -n "$(git status --porcelain ./hacks/cutNewVersions.sh)" ]; then
   git add ./hacks/cutNewVersions.sh
   git commit -m "chore: update versions in cutNewVersions.sh"
 fi
```

Comment:
👀 `if` or `fi`? on line 30 and 33?

Reply:
yeah, I messed this up when I wrote it (or rather when I asked GitHub copilot to write it). `if` to start an if block; `fi` to end it.